falcon/cluster/cluster.py

```diff
@@ -20,6 +21,19 @@
 
 logger = logging.getLogger("falcon")
 
+ConsensusTuple = collections.namedtuple(
+    "ConsensusTuple",
+    [
+        "precursor_mz",
+        "precursor_charge",
+        "mz",
+        "intensity",
+        "retention_time",
+        "cluster",
+        "cluster_size",
```

Comment:
**todo:** I don't think this is used for the algorithm, right? In that case we don't need to store this information.